<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preview</class>
 <widget class="QMainWindow" name="Preview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>768</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PREVIEW - Click on &quot;Generate&quot; to update the gallery</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_BuildingPreview">
      <property name="text">
       <string>... BUILDING PREVIEW ...</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRatio_4_3"/>
   <addaction name="actionRatio_16_9"/>
   <addaction name="actionRotate"/>
   <addaction name="separator"/>
   <addaction name="actionOpenOutputDirectory"/>
  </widget>
  <action name="actionRatio_4_3">
   <property name="icon">
    <iconset resource="../resources/GalleryDesigner.qrc">
     <normaloff>:/icons/images/ratio4_3.png</normaloff>:/icons/images/ratio4_3.png</iconset>
   </property>
   <property name="text">
    <string>Ratio 4:3</string>
   </property>
   <property name="toolTip">
    <string>Ratio 4:3</string>
   </property>
  </action>
  <action name="actionRatio_16_9">
   <property name="icon">
    <iconset resource="../resources/GalleryDesigner.qrc">
     <normaloff>:/icons/images/ratio16_9.png</normaloff>:/icons/images/ratio16_9.png</iconset>
   </property>
   <property name="text">
    <string>Ratio 16:9</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="../resources/GalleryDesigner.qrc">
     <normaloff>:/icons/images/object-rotate-right.png</normaloff>:/icons/images/object-rotate-right.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate the window</string>
   </property>
  </action>
  <action name="actionOpenOutputDirectory">
   <property name="icon">
    <iconset resource="../resources/GalleryDesigner.qrc">
     <normaloff>:/icons/images/folder-image.png</normaloff>:/icons/images/folder-image.png</iconset>
   </property>
   <property name="text">
    <string>Open Output Directory</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open the output directory containing the gallery's files.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/GalleryDesigner.qrc"/>
 </resources>
 <connections/>
</ui>
